pcsx2-qt/ShortcutCreationDialog.ui

```diff
@@ -0,0 +1,287 @@
+           <property name="text">
+            <string>Do not load save state</string>
+           </property>
+           <property name="checked">
+            <bool>true</bool>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="0">
+          <widget class="QRadioButton" name="loadStateIndexToggle">
+           <property name="text">
+            <string>Load save state by slot:</string>
+           </property>
+          </widget>
+         </item>
+         <item row="2" column="0">
+          <widget class="QRadioButton" name="loadStateFileToggle">
+           <property name="text">
+            <string>Load save state from file:</string>
+           </property>
+          </widget>
+         </item>
+         <item row="2" column="2">
+          <widget class="QPushButton" name="loadStateFileBrowse">
+           <property name="text">
+            <string>Browse...</string>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </widget>
+      </item>
+      <item row="9" column="0">
+       <widget class="QLabel" name="saveStateLabel">
+        <property name="text">
+         <string>Save State Option</string>
+        </property>
+       </widget>
+      </item>
+      <item row="3" column="0">
+       <widget class="QCheckBox" name="gameArgsToggle">
+        <property name="text">
+         <string>Game arguments:</string>
+        </property>
+       </widget>
+      </item>
+      <item row="12" column="0" colspan="4">
+       <widget class="QGroupBox" name="customArgsGroup">
+        <property name="title">
+         <string/>
+        </property>
+        <layout class="QGridLayout" name="gridLayout_4">
+         <item row="1" column="0">
+          <widget class="QLineEdit" name="customArgsInput"/>
+         </item>
+         <item row="0" column="0">
+          <widget class="QLabel" name="customArgsInstruction">
+           <property name="text">
+            <string/>
+           </property>
+           <property name="openExternalLinks">
+            <bool>true</bool>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </widget>
+      </item>
+      <item row="0" column="0" colspan="3">
+       <widget class="QCheckBox" name="portableModeToggle">
+        <property name="text">
+         <string>Portable Mode (Stores data in local PCSX2 directory)</string>
+        </property>
+       </widget>
+      </item>
+     </layout>
+    </widget>
+   </item>
+   <item row="6" column="1">
+    <widget class="QDialogButtonBox" name="dialogButtons">
+     <property name="standardButtons">
+      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
+     </property>
+    </widget>
+   </item>
+   <item row="1" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Select Shortcut Type:</string>
```

Comment:
Inconsistent use of semicolons for the headings here.

As a suggestion, here's some consistently formatted headings to use instead:
- Shortcut Type
- Launch Options
- Display Options
- Save State Options
- Custom Arguments